src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -1,128 +1,307 @@
+1. In the **Service Name** field, enter the **Service Name** copied from the connection dialog in [Step 1](#step-1).
+1. Click **Verify**.
+1. In the **VPC** field, enter the ID of the VPC you want to create your endpoint in.
+1. Verify that the subnets are pre-populated.
+1. In the **Security group** section, select the security group you created in [Step 8](#step-8) and uncheck the box for **default** security group.
+1. Click **Create Endpoint**.
 
-      The VPC Endpoint ID displays.
+    The VPC Endpoint ID displays.
 
-1.  Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
+1. Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** dialog.
 
-  </section>
+</section>
 
-  <section class="filter-content" markdown="1" data-scope="aws-cli">
+<section class="filter-content" markdown="1" data-scope="aws-cli">
 
-1.  Substitute the values from the previous steps and run the following AWS CLI command:
+1. Substitute the values from the previous steps and run the following AWS CLI command:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ aws ec2 create-vpc-endpoint --region $REGION \
-    --vpc-id $VPC_ID --subnet-ids $SUBNET_ID1 $SUBNET_ID2 \
-    --vpc-endpoint-type Interface --security-group-ids \
-    $SECURITY_GROUP_ID1 $SECURITY_GROUP_ID2 --service-name \
-    $SERVICE_NAME_PROVIDED_BY_COCKROACH
+    aws ec2 create-vpc-endpoint --region {REGION} \
+                                  --vpc-id {VPC_ID} \
+                                  --subnet-ids {SUBNET_ID1} {SUBNET_ID2} \
+                                  --vpc-endpoint-type Interface \
+                                  --security-group-ids {SECURITY_GROUP_ID1} {SECURITY_GROUP_ID2} \
+                                  --service-name {SERVICE_NAME_PROVIDED_BY_COCKROACH}
     ~~~
 
-1.  Locate the VPC Endpoint ID in the CLI output.
+1. Locate the VPC Endpoint ID in the CLI output.
+
+1. Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** dialog.
 
-1.  Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
+</section>
 
-  </section>
+</section>
 
-### Verify the endpoint ID
+<section class="filter-content" markdown="1" data-scope="serverless">
 
-1.  Paste the Endpoint ID you created into the **VPC Endpoint ID** field.
-1.  Click **Verify**.
-1.  {{ site.data.products.db }} will accept the endpoint request. You can confirm the request acceptance by checking if the status is listed as Available on the Amazon VPC Console **Endpoints** page in your AWS account.
+1. <a name="step-1"></a>Select the region to create a connection in.
 
-### Enable private DNS
+   All of your {{ site.data.products.serverless }} clusters in this region that use PrivateLink will share this connection.
```

Comment:
Not true. Can we please remove this? There can be multiple private endpoints across clusters in an org, and they could be deployed in different VPCs.